spec/src/main/asciidoc/messaging-spec.adoc

```diff
@@ -642,33 +638,33 @@ waiting for them. A message-based application triggering mechanism is
 often used with this style of client. The trigger is typically a
 threshold of waiting messages, etc.
 
-JMS does not provide a mechanism for triggering the execution of a
+Jakarta Messaging does not provide a mechanism for triggering the execution of a
 client. Some providers may supply such a triggering mechanism via their
 administrative facilities.
 
 === Request/reply
 
-JMS provides the JMSReplyTo message header field for specifying the
+Jakarta Messaging provides the JMSReplyTo message header field for specifying the
 Destination where a reply to a message should be sent. The
 JMSCorrelationID header field of the reply can be used to reference the
 original request. See Section “” for more information.
```

Comment:
There are total 9 missing references like this. You can search for empty quotes in the document.

Reply:
There were more, I found 16 missing references. I copied the references from the JMS 2.0 specification. I also checked that the section numbers are correct and they are.